dist/index.html

```diff
@@ -1,33 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leaderboard Project</title>
 <script defer src="bundle.js"></script></head>
+
 <body>
     <header>
         <h1>Leaderboard</h1>
     </header>
     <main>
         <section class="display-section">
-            <div>
+            <div class="recent-score">
                 <h2>Recent scores</h2>
                 <button class="refresh-btn">Refresh</button>
             </div>
-            <table class="table">
-            </table>
+            <div class="result">
+            </div>
         </section>
         <section class="add-section">
             <h2>Add your score</h2>
-            <form>
+            <form method="POST">
                 <input type="text" name="name" class="name" placeholder="your name">
                 <input type="text" name="score" class="score" placeholder="your score">
                 <input type="button" name="submit" class="submit-btn" value="submit">
```

Comment:
[Required] Kindly ensure that the submit button is aligned to the right. As the previous reviewer suggested you should ensure that your design matches the template.